src/microbe_stage/FloatingChunk.cs

```diff
@@ -51,6 +51,8 @@ public class FloatingChunk : RigidBody, ISpawned, ISaveLoadedTracked
     [JsonProperty]
     private float elapsedSinceProcess;
 
+    private string debugName = null!;
```

Comment:
I don't see why this must be its own instance variable.

Reply:
I would much prefer to not need to have special code in each entity related to this feature.